azext_iot/_help.py

```diff
@@ -259,55 +259,94 @@
+                   The expected input file containing device identities should be named 'devices.txt'.
+                   The output log file 'importErrors.log' is empty when import is successful and contains error logs in case of import failure.
+                   Permissions required - Either IoT Hub shared access policy supporting 'Registry Read & Registry Write' OR
+                   an entity with 'IoT Hub Data Contributor' role on the IoT Hub
     long-summary: For more information, see
                   https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities
     examples:
+    - name: Import all device identities from a blob by providing command parameters for
+            input blob container and storage account as well as output blob container and storage account.
+      text: >
+        az iot hub device-identity import -n {iothub_name} --ibc {input_blob_container_name} --isa {input_storage_account_name}
+        --obc {output_blob_container_name} --osa {output_storage_account_name}
     - name: Import all device identities from a blob using an inline SAS uri.
       text: >
         az iot hub device-identity import -n {iothub_name} --ibcu {input_sas_uri} --obcu {output_sas_uri}
     - name: Import all device identities from a blob using a file path which contains SAS uri.
       text: >
         az iot hub device-identity import -n {iothub_name} --ibcu {input_sas_uri_filepath} --obcu {output_sas_uri_filepath}
+    - name: Import all device identities from a blob using system assigned identity that has Storage Blob Data Contributor
+            roles for both storage accounts. The input blob container and storage account as well as output blob container
+            and storage account are provided as parameters to the command
+      text: >
+        az iot hub device-identity import -n {iothub_name} --ibc {input_blob_container_name} --isa {input_storage_account_name}
+        --obc {output_blob_container_name} --osa {output_storage_account_name} --identity [system]
     - name: Import all device identities from a blob using system assigned identity that has Storage Blob Data Contributor
             roles for both storage accounts. The blob container uri does not need the blob SAS token.
       text: >
-        az iot hub device-identity import -n {iothub_name} --ibcu {input_sas_uri} --obcu {output_sas_uri} --auth-type identity --identity [system]
+        az iot hub device-identity import -n {iothub_name} --ibcu {input_sas_uri} --obcu {output_sas_uri} --identity [system]
+    - name: Import all device identities from a blob using user assigned managed identity that has Storage Blob Data Contributor
+            roles for both storage accounts and contributor for the IoT hub. The input blob container and storage account as well as
```

Comment:
Is the part of the statement '...contributor for the IoT hub' accurate?

Reply:
_If_ the user assigned managed identity needs a role against the IoT Hub, wouldn't you need a data plane role like 'IoT Hub Data Contributor' rather than generic 'Contributor'  (a control plane role)?